build-css.js

```diff
@@ -0,0 +1,35 @@
+const sass = require('sass');
```

Comment:
since we already have a script that complies all SCSS to CSS (build-scss.js), let's just repurpose it instead of creating a new one, i.e move all of this code there and remove code that builds `paragon.css` stylesheet

Reply:
Done, thanks

Reply:
Done, thanks